providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -347,6 +349,10 @@ def __init__(
         self.labels = labels or {}
         self.startup_timeout_seconds = startup_timeout_seconds
         self.startup_check_interval_seconds = startup_check_interval_seconds
+        # New parameter startup_timeout_seconds adds breaking change, to handle this as smooth as possible just reuse startup time
+        self.schedule_timeout_seconds = startup_timeout_seconds
+        if schedule_timeout_seconds:
+            self.schedule_timeout_seconds = schedule_timeout_seconds
```

Comment:
This can be made a bit easier:
```suggestion
        self.schedule_timeout_seconds = schedule_timeout_seconds or startup_timeout_seconds
```

Reply:
Changed